src/modules/comments/comments.service.ts

```diff
@@ -0,0 +1,27 @@
+import { UpdateCommentRequestBodyDto } from "./dto/request/update-comment-request-body.dto";
+import { CommentsRepository } from "./comments.repository";
+import { NotFoundError } from "#/shared/errors/not-found-error";
+import { Comment } from "./entities/comment.entity";
+
+export class CommentsService {
```

Comment:
you have service methods for update and delete. Where is for creating and getting all methods ?

Reply:
fixed. I moved methods for updating and deleting comments from blogs service to comments service